commons/src/main/java/io/hyscale/commons/utils/LogProcessor.java

```diff
@@ -59,18 +72,21 @@ public TailLogFile tailLogFile(File logFile, TailHandler handler) {
 		return tailLog;
 	}
 
-	public void readLogFile(File logFile, OutputStream os) {
+	public void readLogFile(File logFile, OutputStream os) throws HyscaleException{
 		readLogFile(logFile, os, DEFAULT_LINES);
 	}
 
 	/*
 	 * Filename, output stream, number of lines
 	 */
-	public void readLogFile(File logFile, OutputStream os, Integer lines) {
-		if (!logFile.exists() || logFile.isDirectory()) {
-			logger.error("File is requird for logs");
+	public void readLogFile(File logFile, OutputStream os, Integer lines)throws HyscaleException {
+		if (logFile == null || !logFile.exists() || logFile.isDirectory()) {
+			logger.error("File is Required for reading.");
```

Comment:
During logging it should be clear that we are unable to read logs. 
This message should be conveyed to user. 
Since the responsibility of the method is reading the log into the file specified, if the file itself is invalid you cannot perform any action , so throw an exception to user.

Reply:
Can you fix this comment, it is still unresolved